enterprise/models.py

```diff
@@ -107,3 +110,57 @@ def __repr__(self):
         Return uniquely identifying string representation.
         """
         return self.__str__()
+
+
+def logo_path(instance, filename):
+    """
+    Delete the file if it already exist and returns the enterprise customer logo image path.
+
+    :param instance: EnterpriseCustomerBrandingConfiguration object
+    :param filename: file to upload
+    :return path: path of image file e.g. enterprise_logo/1/filename
+    """
+    name = os.path.join('enterprise_logo', str(instance.id), filename)
+    fullname = os.path.join(settings.MEDIA_ROOT, name)
+    if os.path.exists(fullname):
```

Comment:
@asadiqbal08 I wonder how would it work with S3 - I don't think `os` module is smart enough to handle that. Probably you should [storage backend object](https://docs.djangoproject.com/en/1.10/topics/files/#storage-objects) instead